content/applications/productivity/knowledge/management.rst

```diff
@@ -1,114 +1,209 @@
+     the URL do not see them unless they can access the inserted content.
+   - Having the Website app is necessary to share an article's URL.
 
-.. image:: management/left-sidebar-cat.png
-   :alt: Categories displayed in the left sidebar of Odoo Knowledge
+.. _knowledge/management/remove:
+
+Removal
+=======
+
+Removing an article involves deleting or archiving it.
+
+Delete an article
+-----------------
+
+Select an article in the sidebar tree and click the :icon:`fa-ellipsis-v` (:guilabel:`ellipsis`)
+icon, then :guilabel:`Send to Trash`. The article is moved to the trash for 30 days before being
+permanently deleted.
+
+Alternatively, click :guilabel:`Search` in the top-left menu. Select an article and click
+:menuselection:`Actions --> Delete --> Delete` to remove the article permanently.
+
+Archive an article
+------------------
+
+Click :guilabel:`Search`, select an article, and click :menuselection:`Actions --> Archive -->
+Archive`.
+
+.. tip::
+   - To restore a deleted article, click :guilabel:`Open the Trash` at the bottom of the sidebar
+     tree, select an article, and click :guilabel:`Restore`. Alternatively, click :guilabel:`Search`
+     in the top-left menu. In the search bar, click :menuselection:`Filters --> Trashed`. Click the
+     article, then :guilabel:`Restore`.
+   - To restore an archived article, click :guilabel:`Search`. In the search bar, click
+     :menuselection:`Filters --> Archived`. Select an article and go to :menuselection:`Action -->
+     Unarchive`.
+
+.. _knowledge/management/retrieve:
+
+Retrieval
+=========
+
+Retrieving Knowledge articles consists of accessing them from various Odoo apps or restoring
+previous versions.
+
+Access articles from various apps
+---------------------------------
+
+Knowledge articles are accessible from the :ref:`form view <studio/views/general/form>` of various
+apps. Click the :icon:`fa-bookmark` :guilabel:`(Knowledge)` icon in the top right corner to open
+the command palette, then choose one of the following search methods:
+
+- :guilabel:`Search for an article`: start typing the text to execute a semantic search that
+  identifies relevant article information.
+- :guilabel:`Advanced Search`: after typing the text in the search bar, click :guilabel:`Advanced
+  Search` to perform a parametric search with options to filter, group, or save articles.
+
+Version history
+---------------
+
+To retrieve a previous version of an article, select it in the sidebar tree and click the
+:icon:`fa-history` (:guilabel:`history`) icon in the top-right menu to open the version history.
+Select a version and click :guilabel:`Restore history`.
+
+.. note::
+   - In the version history, the :guilabel:`Content` tab shows the selected version, while the
+     :guilabel:`Comparison` tab displays the differences between the article's previous and current
+     versions.
+   - By default, archived articles have a 30-day expiration date before being permanently deleted.
```

Comment:
```suggestion
   - By default, deleted articles have a 30-day expiration date before being permanently deleted.
```
Not sure the note is relevant under this heading. Also, it concerns deleted articles as we checked together.